config/routes.rb

```diff
@@ -1,7 +1,7 @@
 Rails.application.routes.draw do
   filter :locale
 
-  root 'main#index'
+  root 'main#welcome'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.